src/doc/unstable-book/src/compiler-flags/min-function-alignment.md

```diff
@@ -0,0 +1,24 @@
+# `min-function-alignment`
+
+The tracking issue for this feature is: https://github.com/rust-lang/rust/issues/82232.
+
+------------------------
+
+The `-Zmin-function-alignment=<align>` flag specifies the minimum alignment of functions for which code is generated.
+The `align` value is interpreted as a power of 2: `-Zmin-function-alignment=5` aligns functions to 32 bytes.
```

Comment:
This seems to be suggesting that we will take the argument of `-Zmin-function-alignment` and perform `2.pow(arg)`? That or it is trying to say 5 rounds up to 8.

Reply:
right, I got things mixed up and this is incorrect, I'll fix that.

The implementation uses `Align::from_bytes` to parse the value. That function rejects any value that is not a power of 2. The GCC and Clang flags also take the value as a number of bytes, but perform a `arg.next_power_of_2()`, so that e.g. `31` is equivalent to `32`. We could of course do that too, but the stricter argument parsing seems right for rust.

Reply:
If we wish to reject inputs like `-Zmin-function-alignment=31` then that seems best to test.